### DISCUSSION TITLE
Can Pagination be used with tables?

### DISCUSSION THREAD

**Discussion by tomasts248** [twbs/bootstrap#37261] [Q&A]
I think most likely not, but to be sure whether ask for it as a feature request. I know there are libraries like Datatables but it is jquery based and far from tiny.

**Comment by WinterSilence** [comment]
other packages not so flexible:
- [wenzhixin/bootstrap-table](https://bootstrap-table.com) still use jQuery
- [jstable/JSTable](https://jstable.github.io) pure JS, but need customize to generate BS styled elements
